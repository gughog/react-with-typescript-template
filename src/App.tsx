import React from 'react';

const App = () => (
  <div className="App">
    Hello, React!
  </div>
);

export default App;
