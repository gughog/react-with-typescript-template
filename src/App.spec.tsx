import React from 'react';
import { render } from '@testing-library/react';
import App from './App';

describe('<App />', () => {
  it('Should match snapshot for App component', () => {
    const { asFragment } = render(<App />);
    expect(asFragment(<App />)).toMatchSnapshot();
  });

  it('Should have "Hello, React!" string on body', () => {
    const { container } = render(<App />);
    expect(container).toHaveTextContent('Hello, React!');
  });
});
