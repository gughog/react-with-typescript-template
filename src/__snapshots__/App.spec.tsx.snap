// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<App /> Should match snapshot for App component 1`] = `
<DocumentFragment>
  <div
    class="App"
  >
    Hello, React!
  </div>
</DocumentFragment>
`;
